extra/onnx_ops.py

```diff
@@ -425,10 +425,10 @@ def _nearest_mode(x_resized: Tensor, nearest_mode: str, x_len):
     elif nearest_mode == "round_prefer_ceil": ret = _round(x_resized, 0.5, "round_up")
     elif nearest_mode == "floor": ret = x_resized.floor()
     elif nearest_mode == "ceil": ret = x_resized.ceil()
-    return ret.clip(0, x_len-1)
+    return ret.clip(0, x_len-1).cast(dtypes.int32)
```

Comment:
is it better to do `cast` before `clip` if we are clipping with int?

Reply:
Trueeee
good call.
the nearest modes should resolve ret to an integer anyways so casting before clip would be good.